content/02.main-text.md

```diff
@@ -19,3 +19,23 @@ We wrote entirely in the open without restrictions on who was welcome to contrib
 Although we requested that some authors participate for their specific expertise, most discovered the manuscript organically through conferences or social media and independently decided to contribute.
 `TODO: confirm "most"`
 To coordinate this effort, we developed a manuscript writing process using the Markdown language, the GitHub software development platform [@url:https://github.com/greenelab/deep-review/], and our new Manubot tool [@url:https://github.com/greenelab/manubot-rootstock @url:https://github.com/greenelab/manubot] for automating manuscript generation.
+
+
+### Authorship
+
+To determine authorship we followed the International Committee of Medical Journal Editors (ICMJE) guidelines and used GitHub to track contributions.
+ICMJE recommends authors substantially contribute to, draft, approve, and agree to be accountable for the manuscript [@url:http://www.icmje.org/recommendations/browse/roles-and-responsibilities/defining-the-role-of-authors-and-contributors.html].
+We acknowledged other contributors who did not meet all four criteria, including contributors who provided text but did not review and approve the complete manuscript.
+`TODO: In Discussion, mention the challenge of determining contributions of potential authors who worked with someone else on the text but did not make their own commits`
+Although these criteria provided a straightforward, equitable way to determine who would be an author, they did not produce a traditionally ordered author list.
+In biomedical journals, the convention is that the first and last authors made the most substantial contributions to the manuscript.
+This convention can be difficult to reconcile in a collaborative effort.
+Using git, we could quantify the number of commits each author made or the number of sentences an author wrote or edited, but these metrics discount intellectual contributions such as discussing primary literature and reviewing pull requests.
+However, there is no objective system to compare and weight the different types of contributions and produce an ordered author list.
+
+To address this issue, we generalized the concept of "co-first" authorship, in which two or more authors are denoted as making equal contributions to a paper.
+We defined four types of contributions [@doi:10.1101/142760], from major to minor, and reviewed the GitHub discussions and commits to assign authors to these categories.
+A randomized algorithm then arbitrarily ordered authors within each contribution category, and we combined the category-specific author lists to produce a traditional ordering.
```

Comment:
For the record, I'm not convinced randomized author order is the right solution. I don't see why the maintainers shouldn't order authors by the extent of their contribution. Of course, this is difficult if there's a large number of authors.

One longterm solution may be to have project members rate each contribution. Assessing contribution in collaborative projects is an unsolved problem. Therefore, I see why the partially randomized ordering is an appealing workaround... but I don't love it.